policy/policy.sample.json

```diff
@@ -0,0 +1,70 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1", # names can be same for multiple tenants thus use uuid.
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants.
```

Comment:
POST: its upto user, if it is provided while post, we can use that, else we can generate it. this is default behavior with openstack or similar platforms.
other verbs: needs PolicyID or Name for lookup.